### DISCUSSION TITLE
Downloading uploaded files via the CP

### DISCUSSION THREAD

**Discussion by anakadote** [statamic/cms#5216] [Troubleshooting]
Is it truely not possible to download files (via the CP) that were uploaded via a frontend form??  I just ran into this issue and I thought that is was something to do with the fact that I'm using an S3 disk, but I found these other posts suggesting that this is an issue regardless of the storage disk:

https://github.com/statamic/cms/pull/4379
https://github.com/statamic/cms/issues/4514
https://github.com/statamic/ideas/issues/669

**CP**
The uploaded file is shown, but there's no way to download it as it's not linked:
<img width="422" alt="Screen Shot 2022-02-07 at 12 53 23 PM" src="https://user-images.githubusercontent.com/2798169/152844369-45cd7f4b-b56c-48f9-872e-579b53a1dd59.png">

**Comment by jasonvarga** [comment]
You've found the issues, so yes, that's correct.

You can go into the assets section and download it from there, in the mean time.

**Reply by anakadote** [reply]
Thanks, @jasonvarga. It's helpful to at least be able to download them from the assets section for the time being.
